src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -660,9 +647,7 @@
 
 		<!-- domain.c: mono_defaults.thread_class -->
 		<!-- FIXME: -->
```

Comment:
What does the `FIXME` refer to here?

Reply:
This was added as Mono was ported over from .NET Framework - so I think it's about the fact that the internal class layout (the context specifically) looks different in Core.
I intentionally left it in because I don't understand this problem space. I think this will need to be revisited for Xamarin as Blazor is probably not exercising this code too much (given that there's really no threading in Blazor).

Reply:
Ideally, we should retain only the fields visible to the runtime.

Reply:
Agree with Zoltan but fields clean up could be done in different PR